packages/quantic/force-app/examples/main/lwc/exampleUseCase/exampleUseCase.html

```diff
@@ -0,0 +1,12 @@
+<template>
+  <template if:true={isSearch}>
+    <c-quantic-search-interface engine-id={engineId}>
+      <slot></slot>
+    </c-quantic-search-interface>
+  </template>
+  <template if:true={isInsight}>
+    <c-quantic-insight-interface insight-id={insightId} engine-id={engineId}>
+      <slot></slot>
+    </c-quantic-insight-interface>
+  </template>
+</template>
```

Comment:
For future's sake, I think you should use 1 render method here and render different templates instead:

https://developer.salesforce.com/docs/component-library/documentation/en/lwc/lwc.create_render

```suggestion
<!-- searchTemplate.html -->
<template>
  <c-quantic-search-interface engine-id={engineId}>
    <slot></slot>
  </c-quantic-search-interface>
</template>

<!-- insightTemplate.html -->
<template>
  <c-quantic-insight-interface insight-id={insightId} engine-id={engineId}>
    <slot></slot>
  </c-quantic-insight-interface>
</template>
```